### PR TITLE
Plane origin

### LINKED ISSUE #189: `origin` function for `Plane`
I am just working on a pull request for `intersecttype(s::Segment{3,T}, p::Plane{3,T}) where {T}` and it has occurred to me that I want to be able to access the fields of `Plane`. For calculating the intersection between a segment(/line/ray) and a plane, the origin and normal of the plane is needed.

@juliohm what do you think about two functions `origin` and `normal` for planes, defined as:
```
function normal(p::Plane{3,T}) where {T}
    p.v × p.w
end
```
and
```
function origin(p::Plane)
    m.p₀
end
```

### PR DESCRIPTION
**Pull request by jackbeagley**
resolves #189

### CODE REVIEW

File: src/primitives/plane.jl [hunk 1]
```diff
@@ -24,3 +24,12 @@ isconvex(::Type{<:Plane}) = true
...
 end
+
+"""
+    origin(p)
+
+Return the origin `p₀` for the plane `p`
+"""
+function origin(p::Plane)
+  p.pₒ
+end
```
Comment (juliohm) [inline src/primitives/plane.jl]:
```suggestion
"""
    origin(plane)

Return the origin of the `plane`.
"""
origin(p::Plane) = p.pₒ
```

**Comment by juliohm** [comment]
@jackbeagley this one is conflicting with the previous PR I just merged. Could you please rebase or start a fresh one?

**Comment by jackbeagley** [comment]
@juliohm all checks passed and ready to merge!

**Comment by juliohm** [comment]
Awesome 💯

**Pull request merged**